src/Pages/styledComponent.js

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+
+export const ParentResizableBox = styled.div`
```

Comment:
styled components???

make a very very simple div. ek box hi karna hai. don't make this complex this is for new joiners

Reply:
.css file use krne pr div k height k liye inlinne css use krna parega